.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+---
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.5b1
```

Comment:
can this versions be taken from requirements file? Otherwise I might forget to update it manually and I'm not sure if dependabot will update it by itself 🤔

Reply:
I could change this to instead use only packages installed in the system, i.e. the ones from our virtualenv. That might be better here, while less consistent with usual usage of pre-commit.

Reply:
https://github.com/dependabot/dependabot-core/issues/1524